### DISCUSSION TITLE
Other platforms

### DISCUSSION THREAD

**Discussion by Eothred** [thrillfall/nextcloud-gpodder#66] [Ideas]
Hi,

I use this with AntennaPod and now installed Kasts on my Linux laptop to see how sync works across devices. For me this is pretty beautiful. However, before I can "brag" about this to my friends I think it would need to be available on apps for other platforms, namely iOS, OSX and Windows (as you probably already have been pondering about).

I see also the integrated Nextcloud Music and Podder apps have been mentioned, which I am sure could be an interesting addition.

I didn't see any discussion on this topic elsewhere (sorry if I missed it), so I thought to maybe ask if there are thoughts/plans/obstacles with getting there? What would be the most sensible apps to target?

**Comment by thrillfall** [comment]
Hi @Eothred,

I am glad synching between different apps works for you. I just realized now this being a use case as you talked about it.

There are no plans to add this to other applications from my side. Anyone who wants to use this in their favorite application can just code it by hirself. There is nothing to do in this code.

**Comment by Eothred** [comment]
Thanks for feedback, and of course for writing the code in the first place!

I assumed this would not be so much on the library side of things but rather thinking of a discussion on where this could be more easily integrated. I suppose whichever podcast applications that already support gpodder.net would be a natural target. Can/could gpodder itself use this kind of instance instead of gpodder.net?

**Reply by thrillfall** [reply]
Yes any client can use this api

**Comment by YehonatanVishna** [comment]
I'm happy to announce there is now a windows client!! 
[link to announcement discussion ](https://github.com/thrillfall/nextcloud-gpodder/discussions/128)
